index.css

```diff
@@ -0,0 +1,6 @@
+header {
+  position: fixed;
+  width: 100%;
+  top: 0;
+  left: 0;
+}
```

Comment:
1. Please kindly install linter for lighthouse configuration
2. Please kindly add style for the rest of the HTML code